rel/app/env.bat.eex

```diff
@@ -2,11 +2,12 @@ if exist "!USERPROFILE!\.livebookdesktop.bat" (
   call "!USERPROFILE!\.livebookdesktop.bat"
 )
 
-set RELEASE_NODE=livebook_app
+if defined LIVEBOOK_NODE set RELEASE_NODE="!LIVEBOOK_NODE!"
```

Comment:
On windows string quotes are sometimes ok but sometimes they aren’t and end up included in the value. Please check if this works correctly if you haven’t.

Reply:
Yeah, even though the lines are ripped from [`rel/server/env.bat.eex`](https://github.com/AviKav/livebook/blob/291ed468ab19599d218c07178fcb4b20d7ae067e/rel/server/env.bat.eex#L6-L7) someone on Windows should probably test it.